circuit/types/boolean/src/helpers/bits_are_zero.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+
+impl<E: Environment> Boolean<E> {
+    /// Asserts that all bits in `bits_le` are zero.
+    #[doc(hidden)]
+    pub fn assert_bits_are_zero(bits_le: &[Boolean<E>]) {
+        let mut sum = Self::constant(false).0;
```

Comment:
I'm wondering what would happen if a smaller field prime were used, like for a different proving system.
Is there an easy way to check the size of `bits_le` outside the circuit to make sure it is less
than the field prime?   If not, just ignore this comment.

Reply:
It looks like @acoglio had the same idea!